dependent-usage.json

```diff
@@ -6,7 +6,7 @@
       "name": "edx-credentials",
       "repository": {
         "type": "git",
-        "url": "git://github.com/edx/credentials"
+        "url": "git://github.com/openedx/credentials"
```

Comment:
Note: this file is automatically generated/updated via a daily Github Action cron workflow so these changes will likely get overridden. I believe these URLs are from whatever the repository field is set as in each repo's package.json file that we're analyzing in the Paragon script. 

That said, we probably also want to change `edx` -> `openedx` in [checkout-dependents.sh](https://github.com/openedx/paragon/blob/master/dependent-usage-analyzer/checkout-dependents.sh#L5-L40) and in the [Github Action workflow definition](https://github.com/openedx/paragon/blob/master/.github/workflows/analyze-dependents.yml#L16).

Reply:
Hey @adamstankiewicz  - I did the fix on those files you mention.

I'm not sure how to go about fixing this particular generated file. Do you know more, or know who I might ask? Is that something that affects any other repos aside from this one?

Reply:
Ah. If I understand properly, https://github.com/openedx/paragon/blob/master/.github/workflows/analyze-dependents.yml should be updated and then the cron will generate this correctly?

Reply:
@sarina Updating analyze-dependents.yml will ensure the Github Action workflow clones the repos directly from the `openedx` organization, but it will have no bearing on the repository fields listed in the auto-generated file as the repository urls listed in the auto-generated file (dependent-usage.json) come from the package.json file in each dependent repos' package.json file, e.g. https://github.com/openedx/frontend-app-learner-portal-enterprise/blob/master/package.json#L87-L90.

Once those `repository` fields are updated in each of Paragon's dependent repos' package.json files (external to Paragon), the auto-generated will start pulling in the correct URLs.

Reply:
gotcha! I'll back out the changes to this file and assume the rest of my work will be sufficient.